### PR TITLE
Refactor Code

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
- we had multiple files for config, merged them into single

### CODE REVIEW

File: src/utils/config.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
-export function getEnv(env = process.env.REACT_APP_ENV) {
-  switch (env) {
+const environment = process.env.REACT_APP_ENV ?? 'prod';
+
+export function getConfig() {
+  const commonConfig = {
```
Comment (harjaapdhillon16) [inline src/utils/config.js:4]: api link for dev and prod are different, please fix it @Megha-Dev-19

**Review by harjaapdhillon16 — CHANGES_REQUESTED** [review]

**Review by harjaapdhillon16 — CHANGES_REQUESTED** [review]
We need to fix the config file

File: src/utils/config.js (hunk 1 above)
Comment (Megha-Dev-19) [reply src/utils/config.js:4]: thanks for informing, I have fixed it

**Review by harjaapdhillon16 — APPROVED** [review]
Looking good

**Pull request merged**